src/status_im/multiaccounts/core_test.cljs

```diff
@@ -0,0 +1,85 @@
+(ns status-im.multiaccounts.core-test
+  (:require [cljs.test :refer-macros [deftest is testing]]
+            [status-im.multiaccounts.core :as ma]))
+
+(def pk "0x0461f576da67dc0bca9888cdb4cb28c80285b756b324109da94a081585ed6f007cf00afede6b3ee5638593674fee100b590318fc7bdb0054b8dd9445acea216ad2")
```

Comment:
I've seen many many people referring to PK as any of these 3 things: public key, private key or the public/private key pair. Could you rename this var to `public-key` or `pub-key` for extra clarity?